lib/std/Build/Step/Compile.zig

```diff
@@ -741,6 +743,8 @@ fn runPkgConfig(compile: *Compile, lib_name: []const u8) !PkgConfigResult {
             try zig_cflags.appendSlice(&[_][]const u8{ "-D", macro });
         } else if (mem.startsWith(u8, arg, "-D")) {
             try zig_cflags.append(arg);
+        } else if (mem.startsWith(u8, arg, wl_rpath_prefix)) {
+            try zig_cflags.appendSlice(&[_][]const u8{ "-rpath", arg[wl_rpath_prefix.len..] });
```

Comment:
hmm, this should use the `compile.addRPath` API instead of C flags I think.

Reply:
@andrewrk: This is basically doing exactly the same as `addRPath`, which is appending `-rpath` in `appendZigProcessFlags` as well.
If we would be using `addRPath` , we'd need to run the `pkg-config` during the configure phase instead of the build phase, which would be weird, as we'd run it twice then